types/openui5/sap.ui.webc.main.d.ts

```diff
@@ -37145,7 +37142,7 @@ declare module "sap/ui/webc/main/TimePicker" {
       /**
        * A Java Script date object to be formatted as string
        */
-      date: Date
+      date: object
```

Comment:
this looks like a regression?

Reply:
...in the original source, I mean

Reply:
multiple times in the same lib

Reply:
Yes, but locally in the source file, not in the generation:

In https://github.com/SAP/openui5/commit/2a5a857e56cdfbe51261c5d6dcf696ae0cd9f40f the return type was manually improved inside the openui5 repo,
but in https://github.com/SAP/openui5/commit/502191ae5bd04a31341a649852285fbc0e9da6c5 the new Web Components version was copied over, reverting the return type to the previous "object".

We need to teach developers to fix those types in the origin repo, not in openui5.

When changing the whitespace, I will also fix this and possibly 1-2 other occurrences.